Makefile.am

```
@@ -173,6 +176,7 @@ include olad/Makefile.mk
 include protoc/Makefile.mk
 include python/Makefile.mk
 include tools/Makefile.mk
+include scripts/Makefile.mk
```

Can you sort these two alphabetically:
```suggestion
include scripts/Makefile.mk
include tools/Makefile.mk
```

---

Done.